gatsby-config.ts

```diff
@@ -70,7 +70,7 @@ const config: GatsbyConfig = {
       options: {
         name: `docs`,
         remote: `https://github.com/opendatahub-io/opendatahub-documentation.git`,
-        branch: `v2.25`,
+        branch: `v3.0`,
```

Comment:
i do not see v3.0 is a vaild branch https://github.com/opendatahub-io/opendatahub-documentation/branches